### PR TITLE
Add contributor license agreement

### PR DESCRIPTION
**Pull request by bonomat**
Adds a Contributor license agreement and update PR template

---
- [x] RTM (Ready To Merge)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
 [Describe your change]
+Got some code for us? Awesome 🎊!
 
+Please include a description of your change & check your PR against this list, thanks!
 
+- [ ] If it resolves a ticket, please add `Resolves #[Issue number]`
+- [ ] Commit messages are meaningful
+- [ ] The code is formatted using `cargo fmt`
+- [ ] The project compiles: `cargo make build`
```
Comment (D4nte) [inline .github/PULL_REQUEST_TEMPLATE.md]: The project and tests runs: `cargo make all`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 
-
-Resolves #[Issue number]
+- [ ] You agree to the [Contributor License Agreement](../CONTRIBUTING.md)
```
Comment (D4nte) [inline .github/PULL_REQUEST_TEMPLATE.md]: ❌ This does not point to the CLA

Comment (bonomat) [reply .github/PULL_REQUEST_TEMPLATE.md]: CLA is in `CONTRIBUTING.md`... I wanted to link to that section directly (e.g. (`../CONTRIBUTING.md#CLA`), but this does not work :(

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (bonomat) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] The project compiles: `cargo make all`
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
+Please include a description of your change & check your PR against this list, thanks!
 
+- [ ] If it resolves a ticket, please add `Resolves #[Issue number]`
+- [ ] Commit messages are meaningful
+- [ ] The code is formatted using `cargo fmt`
+- [ ] The project and tests runs: `cargo make all`
 
-
-Resolves #[Issue number]
+- [ ] You agree to the [Contributor License Agreement](../CONTRIBUTING.md)
```
Comment (D4nte) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] You agree to the [Contributor License Agreement](../CONTRIBUTING.md#contributor-license-agreement)
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -92,3 +92,17 @@ rsqZ
 =wHOK
 -----END PGP PUBLIC KEY BLOCK-----
 ```
+
+## Contributor License Agreement
```
Comment (D4nte) [inline CONTRIBUTING.md]:
```suggestion
## Contributor Licence Agreement
```
Proper English ;)

Comment (bonomat) [reply CONTRIBUTING.md]: Nice try, but I trust wikipedia on that ;) 
https://en.wikipedia.org/wiki/Contributor_License_Agreement

Comment (D4nte) [reply CONTRIBUTING.md]: It's because Wikipedia is written in American...

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -1,9 +1,16 @@
 [Describe your change]
+Got some code for us? Awesome 🎊!
 
+Please include a description of your change & check your PR against this list, thanks!
 
+- [ ] If it resolves a ticket, please add `Resolves #[Issue number]`
+- [ ] Commit messages are meaningful
+- [ ] The code is formatted using `cargo fmt`
+- [ ] The project and tests runs: `cargo make all`
```
Comment (D4nte) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] The project builds and tests run: `cargo make all`
```

**Pull request merged**